vector/src/main/java/im/vector/app/features/settings/legals/ElementLegals.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2021 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.settings.legals
+
+import im.vector.app.R
+import im.vector.app.core.resources.StringProvider
+import im.vector.app.features.discovery.ServerPolicy
+import im.vector.app.features.settings.VectorSettingsUrls
+import javax.inject.Inject
+
+class ElementLegals @Inject constructor(
+        private val stringProvider: StringProvider
+) {
+    /**
```

Comment:
small nit: I don't think this comment/doc adds much, do you think it's needed? renaming `getData` to `getServerPolicies` would avoid the need for a comment

Reply:
Probably but a bit more confusing, since there is no server involved here...